### ISSUE TITLE
Output water balance may not be closed

### ISSUE THREAD

**Issue by dschlaep** [DrylandEcology/rSFSW2#254]
```
with(x, plot(Evaporation_Total_mm_mean - (Evaporation_Soil_Total_mm_mean + Snowloss_mm_mean + Evaporation_InterceptedByVegetation_mm_mean + Evaporation_InterceptedByLitter_mm_mean), pch = 46, ylab = "E - (E(soil) + E(snow) + E(veg) + E(litter))", xlab = "days"))
```

Note: E(surface) is currently not outputted separately (see #253) -- It should be 0 for these simulations.

![pastedgraphic-2](https://user-images.githubusercontent.com/4205019/35169924-55db3bb2-fd2c-11e7-8d1c-da93ce03d5b9.jpg)

##
```
with(x, plot(AET_mm_mean- (Transpiration_Total_mm_mean + Evaporation_Total_mm_mean), pch = 46, ylab = "AET - (T + E)", xlab = "days"))
```
![pastedgraphic-1](https://user-images.githubusercontent.com/4205019/35169892-3553519a-fd2c-11e7-9f1f-21c9e9769ada.jpg)

**Comment by dschlaep** [comment]
x-axes labels are `Sites` and y-axes are in units of `mm year-1`

**Comment by dschlaep** [comment]
This error got fixed with the fixes to DrylandEcology/SOILWAT2#133 and DrylandEcology/rSOILWAT2#99:
```
library("RSQLite")
con <- dbConnect(SQLite(), "tests/test_data/0_ReferenceOutput/dbTables_Test4_AllOverallAggregations_snow_v2.5.6.sqlite3")
x <- dbReadTable(con, "aggregation_overall_mean")

# Checks from this issue:
# Check: Etotal == (E(soil) + E(snow) + E(veg) + E(litter))
with(x, all.equal(Evaporation_Total_mm_mean, (Evaporation_Soil_Total_mm_mean + Snowloss_mm_mean + Evaporation_InterceptedByVegetation_mm_mean + Evaporation_InterceptedByLitter_mm_mean)))

# Check: AET == (Ttotal + Etotal)
with(x, all.equal(AET_mm_mean, (Transpiration_Total_mm_mean + Evaporation_Total_mm_mean)))

dbDisconnect(con)
```

> TRUE
> TRUE
